deploy/crd.yaml

```diff
@@ -30,6 +30,40 @@ spec:
     kind: Database
     plural: databases
   scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - users
+            - server
+            - serverType
+          properties:
+            users:
+              type: array
+              items:
+                type: object
+                required:
+                - name
+                - role
+                properties:
+                  name:
+                    type: string
+                  password:
+                    type: string
+                  role:
+                    type: string
+                    enum:
+                      - 'admin'
+                      - 'read'
+            server:
+              type: string
+            serverType:
+              type: string
+              enum:
+                 - 'postgres'
+  subresources:
```

Comment:
why did you added this ? right now we don't support CustomResourceSubResources.

Reply:
will remove subresources part